_raml/apps/product/schema/get_product_productid.schema.json

```diff
@@ -3,93 +3,9 @@
   "title": "Get Product ProductId Schema",
   "type": "object",
```

Comment:
i assume you have to write type=object of [product](page:apps-data-...) here (below "title") and drop the "properties" attribute completely since the properties are described somewhere else now.